clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListAllTags.java

```diff
@@ -53,7 +53,7 @@ public void handle() {
       exitWithError(exp.getMessage());
     }
 
-    String all = String.join(",", tags);
+    String all = tags.length == 0 ? "No tags exists." : String.join(",", tags);
```

Comment:
"No tags" is still a plural.

```suggestion
    String all = tags.length == 0 ? "No tags exist." : String.join(",", tags);
```